features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/org.wso2.carbon.identity.event.server.feature.default.json

```diff
@@ -9,6 +9,7 @@
     "POST_NON_BASIC_AUTHENTICATION"
   ],
   "identity_mgt.events.schemes.'account.lock.handler'.properties.enable": false,
+  "identity_mgt.events.schemes.'account.lock.handler'.properties.lock.on.max.failed.attempts.enable": false,
   "identity_mgt.events.schemes.'account.lock.handler'.properties.'login.fail.timeout.ratio'": "2",
```

Comment:
We no longer needed identity_mgt.events.schemes.'account.lock.handler'.properties.enable?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/3648/commits/1e2969567fb074b461d36166d387bdf8ba67fd1f